CHANGELOG.md

```diff
@@ -122,7 +123,6 @@ Changes
 * [UI]: Fixed bug causing issues with saving Line List templates.
 * [UI]: Fixed bug when selecting all samples on the project samples page would not add them to cart.
 * [Developer]: Updated to latest version of NodeJS LTS (12.13.0) and Yarn (v1.19.1).
-* [UI]: Add analyses queued and running counts to analyses listing pages, cart page, and pipeline launch page.
```

Comment:
You deleted this but did not move it up.  Be easier to merge later if you move it I think.

Reply:
This should be fixed in the latest development merge.